src/realm/realmReducer.js

```diff
@@ -18,6 +18,10 @@ const initialState = {
   filters: [],
   emoji: {},
   videoChatProvider: null,
+  realmAllowMessageEditing: true,
+  realmAllowMessageDeleting: true,
+  realmMessageContentEditLimit: 10,
+  realmMessageContentDeleteLimit: 10,
```

Comment:
Where do these values come from?

Reply:
@WesleyAC  I think this is the initial value fixed when a user opens any new org in zulip( that's why I kept this,  although I had a little doubt  to keep it as `initialState` value 🤔 )